cli/tests/integrations/test_marathon.py

```diff
@@ -388,6 +386,8 @@ def test_update_invalid_request():
     assert returncode == 1
     assert stdout == b''
     stderr = stderr.decode()
+    # TODO (tamar): this becomes 'Error: App '/{' does not exist\n"'
+    # in Marathon 0.11.0
```

Comment:
Let's avoid TODOs in committed code. File a Jira if we need to change this. Thanks!